management/src/test/groovy/io/micronaut/management/endpoint/health/HealthEndpointSpec.groovy

```diff
@@ -103,7 +106,7 @@ class HealthEndpointSpec extends Specification {
 
     void "test the beans are not available with all disabled"() {
         given:
-        ApplicationContext context = ApplicationContext.run(['endpoints.all.enabled': false])
+        ApplicationContext context = ApplicationContext.run(['endpoints.all.enabled': StringUtils.FALSE])
```

Comment:
Not sure why this is better, but there's 3 other places we can change if we want to be consistent?

Lines 67, 81 and 95